powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java

```diff
@@ -562,52 +562,6 @@ void getEciesDecryptorTest() throws CryptoProviderException, GenericCryptoExcept
         assertArrayEquals(requestData.getBytes(StandardCharsets.UTF_8), decryptedData);
     }
 
-    @Test
-    void upgradeTest() throws CryptoProviderException, GenericCryptoException, InvalidKeyException, InvalidKeySpecException, EciesException, IOException, PowerAuthClientException {
-        KeyPair clientEphemeralKeyPair = keyGenerator.generateKeyPair();
-        KeyPair deviceKeyPair = activation.generateDeviceKeyPair();
-        String activationIdentity = UUID.randomUUID().toString();
-        String activationOTP = "00000-00000";
-        byte[] nonceDeviceBytes = activation.generateActivationNonce();
-        byte[] cDevicePublicKeyBytes = activation.encryptDevicePublicKey(deviceKeyPair.getPublic(), clientEphemeralKeyPair.getPrivate(), config.getMasterPublicKey(),
-                activationOTP, activationIdentity, nonceDeviceBytes);
-        byte[] signature = activation.computeApplicationSignature(activationIdentity, nonceDeviceBytes, cDevicePublicKeyBytes, Base64.getDecoder().decode(config.getApplicationKey()),
-                Base64.getDecoder().decode(config.getApplicationSecret()));
-        byte[] ephemeralPublicKeyBytes = keyConvertor.convertPublicKeyToBytes(clientEphemeralKeyPair.getPublic());
-        com.wultra.security.powerauth.client.v2.CreateActivationResponse createResponse = powerAuthClient.v2().createActivation(config.getApplicationKey(), config.getUserV31(), activationIdentity,
```

Comment:
I do not know how to rewrite this test if `powerAuthClient.v2()` has been dropped.